packages/api/src/routes/medical/patient.ts

```diff
@@ -339,6 +342,11 @@ router.post(
     const dateFrom = parseISODate(getFrom("query").optional("dateFrom", req));
     const dateTo = parseISODate(getFrom("query").optional("dateTo", req));
     const type = getFrom("query").optional("conversionType", req);
+    const generateAiBrief = getFromQueryAsBoolean("generateAiBrief", req) ?? false;
+    const isAiBriefFeatureFlagEnabled = await isAiBriefEnabledForCx(cxId);
+    if (!isAiBriefFeatureFlagEnabled && generateAiBrief) {
+      throw new BadRequestError("Contact Metriport to enable the AI Brief feature.");
+    }
```

Comment:
Moved the check + added log and capture, so we're notified if this happens

Reply:
Sounds good. I suggested to have it as part of the command, though, so we minimize logic on the route and have most of the logic on the command. It means that, whenever someone calls that command, it will run all the logic there - and we don't need to worry whether we're missing stuff that was only coded on the route.

In the onion pattern, the routes' sole reason for existence is to convert from HTTP into app-specific code and vice-versa.